ui/components/app/metamask-translation/README.mdx

```diff
@@ -0,0 +1,36 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import MetaMaskTranslation from '.';
+
+# MetaMask Translation
+
+MetaMaskTranslation is a simple helper Component for adding full translation
```

Comment:
I don't think `Component` needs to be uppercase

Reply:
resolved